code/code/cmd/cmd_zonefile.cc

```diff
@@ -524,20 +438,14 @@ void TBeing::doZonefile(const sstring & tStArg)
   if (tStArg.empty()) {
     sendTo("Syntax:\n\r");
     sendTo("zonefile save : Save current zonefile status.\n\r");
-    sendTo("zonefile load : Load current zonefile status.\n\r");
     sendTo("zonefile new <n> <builder - zone title>: Split <n> rooms off the last block, creating a new zone named <builder - zone title>.\n\r");
-#if 0
-    sendTo("zonefile modify <field> SEE HELP FILE\n\r");
-#endif
   }
 
   std::string cmd = tStArg.word(0);
   std::string rest = tStArg.substr(tStArg.find(' ') + 1);
 
   if (is_abbrev(cmd, "save")) {
     doSaveZoneFile(this, rest);
-  } else if (is_abbrev(cmd, "load") && getName() == "Lapsos") {
```

Comment:
Why would someone have done that in the first place?

Reply:
Perhaps they didn't have a test instance?